VMEncryption/main/Common.py

```diff
@@ -20,12 +20,15 @@
 class CommonVariables:
     utils_path_name = 'Utils'
     extension_name = 'AzureDiskEncryptionForLinux'
-    extension_version = '1.1.0.40'
+    # Change based on test pipeline used
+    test_extension_name = 'AzureDiskEncryptionForLinuxTest1'
+    extension_version = '1.1.0.41'
     extension_type = extension_name
     extension_media_link = 'https://amextpaas.blob.core.windows.net/prod/' + extension_name + '-' + str(extension_version) + '.zip'
     extension_label = 'Azure Disk Encryption For Linux VMSS'
     extension_description = extension_label
     extension_shim_filename = "extension_shim.sh"
+    test_extension_publisher = 'Microsoft.Azure.Security.Edp.'
```

Comment:
Are test_extension_name and test_extension_publisher leftovers, or intentionally included?

Reply:
Yes it is intentional. When any of us is deploying a test extension, he should change the value here depending on the test pipeline used. This will ensure extension logs from pivot root env are preserved. Let me know if it sound ok.